examples/stable/mysql/mysql-blueprint.yaml

```diff
@@ -28,9 +28,9 @@ actions:
         - pipefail
         - -c
         - |
-          s3_path="/mysql-backups/{{ .StatefulSet.Namespace }}/{{ index .Object.metadata.labels "release" | toString }}/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}/dump.sql.gz"
+          s3_path="/mysql-backups/{{ .StatefulSet.Namespace }}/{{ index .Object.metadata.labels "app.kubernetes.io/instance" }}/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}/dump.sql.gz"
```

Comment:
I am not sure about the consequences of remove `| string` here. Otherwise looks good.